UnitsNet.Tests/QuantityIFormattableTests.cs

```diff
@@ -84,10 +85,10 @@ public void FormattingUsesSuppliedLocale()
         public void FormatStringWorksWithSuppliedLocale()
         {
             Temperature t = Temperature.FromDegreesCelsius(2012.1234);
-            CultureInfo c = CultureInfo.CreateSpecificCulture("de-CH");
+            CultureInfo c = new CultureInfo("de-CH", false);
 
             FormattableString f = $"{t:g}";
-            Assert.Equal("2'012.12 °C", f.ToString(c));
+            Assert.Equal("2" + c.NumberFormat.NumberGroupSeparator + "012" + c.NumberFormat.NumberDecimalSeparator + "12 °C", f.ToString(c));
```

Comment:
I would hard code the expected string here instead of dynamically looking it up. Same above.

Reply:
I changed this just because of the problem that showed up yesterday: The characters Windows uses for these members change. And so any Windows Update would break the test. It could also be that the values are different on another platform (i.e when running the tests under linux).

Reply:
Ok, although I don't expect Windows to change this back and forth frequently, it's fine to keep it like this too.